.idea/compiler.xml

```diff
@@ -26,7 +26,7 @@
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel target="1.8">
-      <module name="ews-java-api" target="1.8" />
+      <module name="ews-java-api" target="1.7" />
```

Comment:
Didn't realize these IntelliJ files were checked in. I can revert this if anyone thinks we should/need to.